crates/ibc/src/mock/header.rs

```diff
@@ -71,9 +71,18 @@ impl MockHeader {
     }
 
     pub fn new(height: Height) -> Self {
-        Self {
-            height,
-            timestamp: Timestamp::now(),
+        cfg_if::cfg_if! {
+            if #[cfg(any(test, feature = "mocks"))]  {
+                Self {
+                    height,
+                    timestamp: Timestamp::now(),
+                }
+            } else if #[cfg(any(test,feature = "mocks-no-std"))] {
```

Comment:
```suggestion
            } else {
```
Improves readability, and ensures that we always compile something in.